hack/verify-generated-clientsets.sh

```diff
@@ -8,7 +8,7 @@ source "${OS_ROOT}/hack/lib/init.sh"
 
 cd "${OS_ROOT}"
 echo "===== Verifying Generated Client sets ====="
-output=$(VERIFY=true ${OS_ROOT}/hack/update-generated-clientsets.sh 2>&1)
+output=$(VERIFY=true ${OS_ROOT}/hack/update-generated-clientsets.sh)
```

Comment:
we do, this was just debugging, because "sometimes" I get error 255 in travis....